authkey_test

```diff
@@ -0,0 +1 @@
+tskey-auth-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
I'm not sure where else to put this but this is only relevant for the test. :sweat_smile: